jobs/src/main/scala/dcos/metronome/jobrun/impl/JobRunExecutorActor.scala

```diff
@@ -131,6 +127,17 @@ class JobRunExecutorActor(
     startingDeadlineTimer = None
   }
 
+  def tasksFromTaskTracker(): Iterable[JobRunTask] = {
+    taskTracker.appTasksLaunchedSync(runSpecId).collect {
+      // FIXME: we do currently only allow non-resident tasks. Since there is no clear state on
```

Comment:
lets remove this FIXME.   I'm fine with noting that we only work with Ephemeral...   Any change to this should be represented in a JIRA not a FIXME note.